miso-web/src/main/webapp/scripts/hot_pool.js

```diff
@@ -82,6 +82,74 @@ HotTarget.pool = {
                 ids : items.map(Utils.array.getId).join(',')
               });
         }
-      }, HotUtils.printAction('pool'), ],
+      },
+      {
+        name : "Create Order",
+        action : function(pools) {
+          var platformTypes = Utils.array.deduplicateString(pools.map(function(
+              pool) {
+            return pool.platformType;
```

Comment:
Could this be changed to use the active PlatformTypes? Or perhaps we could just delete the `Solexa 1G Genome Analyzer` and other unused friends from our db...